lib/dao.js

```diff
@@ -2669,6 +2669,19 @@ DataAccessObject.updateAll = function(where, data, options, cb) {
   this.applyScope(query);
   this.applyProperties(data);
 
+  var doValidate = undefined;
+  if (options.validate === undefined) {
+    if (Model.settings.validateUpdate === undefined) {
```

Comment:
`update` and `updateAll` are aliases. As the jsdoc for the function says:
```
/**
 * Update multiple instances that match the where clause
 *
 * Example:
 *
 *```js
 * Employee.update({managerId: 'x001'}, {managerId: 'x002'}, function(err) {
 *     ...
 * });
 * ```
 *
 * @param {Object} [where] Search conditions (optional)
 * @param {Object} data Changes to be made
 * @param {Object} [options] Options for update
 * @param {Function} cb Callback, called with (err, info)
 */
```

I think we it would be the best to allow the user specify either `validateUpdate` or `validateUpdateAll` flag. In that case we need to take care of both the flags.